packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -1720,8 +1720,25 @@ export class ApexDebug extends LoggingDebugSession {
 
       // log to console and notify client
       this.logEvent(message);
+      let reason = '';
+
+      // if breakpointid is found in exception breakpoint cache
+      // set the reason for stopped event to that reason
+      if (message.sobject.BreakpointId) {
+        const cache: Map<
```

Comment:
@DatGuyJonathan not sure if this is what you had in mind for getting the exception type. The other way of easily getting it is by parsing `message.sobject.Description`. 

Another option would be to inverse the exception breakpoint cache map so that breakpoint id are the keys and the type is the value but this would require changing the exception breakpoint reconciliation.